pkg/cmd/start/bootstrap.go

```diff
@@ -52,6 +53,10 @@ import (
 	esv1 "github.com/openshift/elasticsearch-operator/apis/logging/v1"
 )
 
+// We should avoid that users unknowingly use a vulnerable TLS version.
+// The defaults should be a safe configuration.
+const defaultMinTLSVersion = tls.VersionTLS12
```

Comment:
Would it make sense to introduce a way to change the minTLS version?

Reply:
Yes definitely, may b we can do it by adding a configuration from cli argument? May b u can point me on this?

Reply:
seems you already figured it out :)